install.sh

```diff
@@ -56,8 +56,12 @@ if ! sudo apt-get update; then
     die_cmd "Failed to update APT cache."
 fi
 
-if ! sudo apt-get install gpg wget; then
-    die_cmd "Failed to check if needed packages are installed."
+if ! ( wget --version > /dev/null 2>&1 ||  sudo apt-get install --mark-auto wget ); then
+    die_cmd "Failed to install wget."
+fi
+
+if ! ( gpg --version > /dev/null 2>&1 ||  sudo apt-get install --mark-auto gpg ); then
+    die_cmd "Failed to install gpg."
```

Comment:
Could this be moved to one command so the entire `apt install` process is done in one command? I think if we just run something like `apt-get satisfy gpg wget` we should be able to do this in one command too, and the packages will still be marked as auto installed if they were previously (or even if newly installed, so they can be removed after this script is done if they're not needed by anything).